src/main/java/com/hbm/items/food/ItemEnergy.java

```diff
@@ -215,62 +216,63 @@ public ItemStack onItemRightClick(ItemStack stack, World world, EntityPlayer pla
 	@SideOnly(Side.CLIENT)
 	public void addInformation(ItemStack p_77624_1_, EntityPlayer p_77624_2_, List list, boolean p_77624_4_) {
 		if(this == ModItems.can_smart) {
-			list.add("Cheap and full of bubbles");
+			list.add(I18nUtil.resolveKey("item.can_smart.desc"));

```

Comment:
At that point you might as well can the `if` checks entirely and use `this.getUnlocalizedName() + ".desc"` for everything